models/event-videos.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "event.videos",
+  "type": "array",
+  "description": "Describes an array of video objects that are linked to an event.",
+  "examples": [
+    {
+      "id": "b504cf44-9ab8-4641-9934-38d1cc67242c",
+      "url": "https://www.youtube.com/?v=ewr53d",
+      "copyrightHolder": "publiq"
+    },
+    {
+      "id": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
+      "url": "https://www.vimeo.com/54fdrg"
+    }
+  ],
+  "items": {
+    "$ref": "./video.json"
```

Comment:
Nit: Because this is not a resource that exists on its own I would inline the schema here, even though you would have to duplicate it. Duplication is not necessarily bad here, you also did it for example in `place-add-video.json` and `event-add-video.json` where you duplicated the schema for the POST payload. And this keeps the overview of schema files a bit clearer. Because we might introduce the concept of a video later on in another context and then this "video.json" will become ambiguous.

Reply:
Good point I inlined the video objects for place and event.